Mathlib/Data/List/Basic.lean

```diff
@@ -198,10 +198,6 @@ theorem map_subset_iff {l₁ l₂ : List α} (f : α → β) (h : Injective f) :
 theorem append_eq_has_append {L₁ L₂ : List α} : List.append L₁ L₂ = L₁ ++ L₂ :=
   rfl
 
-@[deprecated (since := "2024-03-24")] alias append_eq_cons_iff := append_eq_cons
-
-@[deprecated (since := "2024-03-24")] alias cons_eq_append_iff := cons_eq_append
-
```

Comment:
Are these names being reused? Or is there another reason for dropping these already?

Reply:
I ended up restoring the `_iff` ... Sorry for the churn.